src/nectarchain/dqm/charge_integration.py

```diff
@@ -45,26 +45,37 @@ def ConfigureForRun(self, path, Chan, Samp, Reader1):
         self.image_ped = []
         self.peakpos_ped = []
 
+        
+
 
     def ProcessEvent(self, evt, noped):
         #print("test", evt.r0.tel[0].waveform[0])
+        self.pixelBAD = evt.mon.tel[0].pixel_status.hardware_failing_pixels
         pixel = evt.nectarcam.tel[0].svc.pixel_ids
-        pixel21 = np.arange(0,21,1,dtype = int)
-        pixel = list(pixel)
-        pixel21 = list(pixel21)
-        pixels = np.concatenate([pixel21,pixel])
+        
```

Comment:
I do not understand the relationship of this change to the upgrade to ctapipe 0.19

Reply:
This is actually part of #63 and should not be duplicated here. @hashkar , I will remove this part.